piccolo_api/shared/middleware/junction.py

```diff
@@ -15,12 +17,14 @@ def __init__(self, *routers: Router) -> None:
     async def __call__(self, scope: Scope, receive: Receive, send: Send):
         for router in self.routers:
             try:
-                asgi = await router(scope, receive=receive, send=send)
+                response: t.Any | None = await router(
```

Comment:
Just a tip. In future, you must be careful to use syntax that will satisfy the tests in `Py3.7`, `Py3.8`, `P3.9`, and `Py3.10`. I run your branch and everything should be fine, just remove the `import typing as t` from file because is unused.

Reply:
Yeah, I'll be careful. Done the changes. Hope everything will be good now. Please have a look and merge it.

Reply:
Great. You have to wait @dantownsend for merging.